itest/list_exclude_test.go

```diff
@@ -76,7 +75,10 @@ var excludedTestsWindows = []string{
 	// more investigation is needed.
 	"channel force close-anchor restart",
 	"channel force close-simple taproot restart",
-}
+}, extractNames(
```

Comment:
hmmm, how about we add this to L103? Seems easier this way, also keep this `excludedTestsWindows` simple
```go
                // The name for a bubtest case has the format
		// "subtestName-caseName", to exclude a whole subtest, we match
		// against "subtestName".
		name := strings.Split(tc.Name, "-")[0]
		if excludedSet.Contains(name) {
			excludedSet.Remove(name)

			continue
		}
```

Reply:
This might give us false positives with names like `channel force close-anchor restart`...

Reply:
"channel force close" is also a prefixed test.

In `itest/list_on_test.go`:
```go
        allTestCases = appendPrefixed(
                "channel force close", allTestCases, channelForceCloseTestCases,
        )
```

Removing "channel force close" from `excludedSet` should not be an issue, since it is not present there. It would be a no-op.

Reply:
Hmm, I actually prefer the original approach (`extractNames`). It is explicit and clear that this is a group of prefixed tests.